src/dapp/README.md

```diff
@@ -265,7 +265,9 @@ The `--verify` flag verifies the contract on etherscan (requires `ETHERSCAN_API_
 ## Configuration
 
 The commands of `dapp` can be customized with environment variables or flags.
-These variables can be set at the prompt or in a `.dapprc` file.
+These variables can be set at the prompt or in a `.dapprc` file. 
+
+Below is a non-comprehensive list of some common configuration options (more can be found in the sourcecode):
```

Comment:
Which variables are missing? Can we just add them to the list here instead of directing people to the source?

Reply:
> Which variables are missing? Can we just add them to the list here instead of directing people to the source?

Well there's like a *bunch* of variables that you can customize that are typically set with flags (verbosity, etc) that I'm not sure you guys want to encourage ppl using as config options? I can add if you do.

Reply:
hmmmmm, not sure which ones you mean, but I think I'm fine having all of them here. It's probably also worth linking to the [environment variables](https://github.com/dapphub/dapptools/blob/master/src/hevm/README.md#environment-variables) section in the hevm readme (or maybe even just adding them here?).